spiketools/tests/spatial/test_occupancy.py

```diff
@@ -1,12 +1,24 @@
 """Tests for spiketools.spatial.occupancy"""
 
 from spiketools.spatial.occupancy import *
+import numpy as np
 
 ###################################################################################################
 ###################################################################################################
 
-def test_compute_spatial_bin_edges():
-    pass
+def test_compute_spatial_bin_edges(position, bins):
```

Comment:
In general, test functions don't take inputs (sometimes they do, but these are special test objects defined in `conftest.py`. In this case (for now at least) the function shouldn't take any inputs, and you should define some test data within the function. 

Check `spiketools/tests/measures/test_measures.py` for some examples.